src/containers/App/App.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import './App.css'
 import { Button, ButtonType, ButtonSize } from 'components'
 import { Checkbox, CheckboxType } from 'components/Checkbox'
+import { FormGroup } from 'components/FormGroup'
```

Comment:
Actually, should be like this, not the change I suggested earlier
```suggestion
import { Checkbox, CheckboxType, FormGroup } from 'components'
```

Reply:
Cool beans

Reply:
Also needed to add:
```
export * from './Checkbox'
export * from './FormGroup'
```
to `src/components/index.ts`. 

It's not passing checks. I'll merge it in 🚀